### PR TITLE
Feauture/update map

### PR DESCRIPTION
**Pull request by GiftLanga**

### CODE REVIEW

File: waterlog/src/Pages/Home.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ class HomeComponent extends Component {
         <AlertsMap />
         <Wastage />
         <p className="home-text">Tank Levels</p>
-        <TankLevels />
+        {/*<TankLevels />*/}
```
Comment (CilliersN) [inline waterlog/src/Pages/Home.js]: gc

Comment (FredJEhlers) [inline waterlog/src/Pages/Home.js]: GC

**Review by CilliersN — CHANGES_REQUESTED** [review]

File: waterlog/src/Pages/Home.js (hunk 1 above)
Comment (CilliersN) [reply waterlog/src/Pages/Home.js]: Also, if you remove < TankLevels/ > be sure to remove its import if not used anymore
`import TankLevels from './../Components/TankLevels';`

Comment (Tebogo-Seshibe-Retro) [inline waterlog/src/Pages/Home.js]: Why comment out this component?

**Comment by CilliersN** [comment]
**in your Map.js file**
its good practice in react to multiline if more than two imports:
```
import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
import { sensorFaultIcon, sensorOkIcon } from './../icons/MapIcons';
```

----------------
```
import { Map, 
             TileLayer, 
             Marker, 
             Popup 
} from 'react-leaflet';
import { sensorFaultIcon, sensorOkIcon } from './../icons/MapIcons';
```


just apply correct indentation

File: waterlog/src/Pages/Home.js (hunk 1 above)
Comment (GiftLanga) [reply waterlog/src/Pages/Home.js]: The code has errors, We are still to fix them

Comment (GiftLanga) [reply waterlog/src/Pages/Home.js]: The component has errors, the person responsible for it is fixing it

Comment (CilliersN) [reply waterlog/src/Pages/Home.js]: please add a TODO with description

File: waterlog/src/Components/Map.js [hunk 2]
```diff
@@ -1,13 +1,23 @@
 import React, { Component } from 'react';
-import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
-import { sensorFaultIcon, sensorOkIcon } from './../icons/MapIcons';
+import { Map, 
+        TileLayer, 
+        Polyline, 
+        CircleMarker, 
+        Popup } from 'react-leaflet';
```
Comment (CilliersN) [inline waterlog/src/Components/Map.js]: `} from 'react-leaflet';` on a newline

Comment (GiftLanga) [reply waterlog/src/Components/Map.js]: I moved it to a new line

File: waterlog/src/Pages/Home.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ class HomeComponent extends Component {
         <AlertsMap />
         <Wastage />
         <p className="home-text">Tank Levels</p>
-        <TankLevels />
+        {/*TODO <TankLevels />*/}
```
Comment (grootendorste) [inline waterlog/src/Pages/Home.js]: I would rather just take this out. I mean if you wrote up a ticket for it, or a tester sees it, trust me, you'll be notified of it. This is just dead code to me. But it's your call

Comment (GiftLanga) [reply waterlog/src/Pages/Home.js]: I will remove it, since I am not the one implementing it

File: waterlog/src/Pages/Home.js (hunk 1 above)
Comment (GiftLanga) [reply waterlog/src/Pages/Home.js]: I will remove the comment since it does not affect my feature

**Review by CilliersN — APPROVED** [review]

**Pull request merged**